doc/src/sgml/ref/create_materialized_view.sgml

```diff
@@ -58,8 +58,79 @@ CREATE [ INCREMENTAL ] MATERIALIZED VIEW [ IF NOT EXISTS ] <replaceable>table_na
     <term><literal>INCREMENTAL</literal></term>
     <listitem>
      <para>
-      If specified, a materialized view enables incremental view maintenance.
-      You can replace only the contents of a materialized view, which based rows are changed.
+      If specified, some triggers and other objects are automatically created
+      so that the rows of the materialized view is immediately updated when
+      base tables of the materialized view are updated. In general, this
+      allows faster update of the materialized view at a price of slower
+      update of the base tables because the triggers will be invoked.
+     </para>
+     <para>
+      There are restrictions of query definitions allowed to use this
+      option. Followings are allowed query definitions:
+      <itemizedlist>
+       <listitem>
+        <para>
+         Inner joins (including self-joins).
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Some of aggregations (count, sum, avg) without HAVING clause.
+        </para>
+        </listitem>
+      </itemizedlist>
+
+      Prohibited queries with this option include followings:
+      <itemizedlist>
+       <listitem>
+        <para>
+         Outer joins.
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Subqueries.
+        </para>
+        </listitem>
+       <listitem>
+        <para>
+         Aggregations other than count, sum or avg.
```

Comment:
Now, we are also supporting min/max aggregate.

Reply:
Oops. I will add min/max.